lib/py/src/transport/THttpClient.py

```diff
@@ -111,8 +111,6 @@ def open(self):
                                                      timeout=self.__timeout)
         elif self.scheme == 'https':
             self.__http = http.client.HTTPSConnection(self.host, self.port,
-                                                      key_file=self.keyfile,
-                                                      cert_file=self.certfile,
                                                       timeout=self.__timeout,
```

Comment:
> Deprecated since version 3.6: key_file and cert_file are deprecated in favor of context. Please use ssl.SSLContext.load_cert_chain() instead, or let ssl.create_default_context() select the system’s trusted CA certificates for you.
> 
> The check_hostname parameter is also deprecated; the ssl.SSLContext.check_hostname attribute of context should be used instead.

https://docs.python.org/3/library/http.client.html

Where is the second part of that change?

Reply:
> please use ssl.SSLContext.load_cert_chain() instead, or let ssl.create_default_context() select the system’s trusted CA certificates for you.

Copypasted too much. I meant that part actually.

Reply:
Already here https://github.com/apache/thrift/blob/7734c393ed0f0632c658c05e33a4d6592cf2912c/lib/py/src/transport/THttpClient.py#L117

Reply:
But already here https://github.com/apache/thrift/blob/8570d1b45ee6e35573cf453861a83cc5e11c5ceb/lib/py/src/transport/THttpClient.py#L66-L67

Reply:
https://github.com/vitaly-zdanevich/thrift/blob/8570d1b45ee6e35573cf453861a83cc5e11c5ceb/lib/py/src/transport/TSSLSocket.py#L177-L179